scripts/publish.sh

```diff
@@ -32,8 +32,9 @@ echo "Publish Latest? ${PUBLISH_LATEST}"
 
 # this code currenly works because DockerHub returns the only last 100 tags and as of March 2023 we only have 64
 # and it should keep working because dockerhub returns the latest tags first
-docker_hub_image_tags=$(curl -s -S 'https://registry.hub.docker.com/v2/repositories/amazon/aws-for-fluent-bit/tags/?page=1&page_size=250' | jq -r '.results[].name')
-tag_array=(`echo ${docker_hub_image_tags}`)
+public_ecr_image_tags_token=$(curl -k https://public.ecr.aws/token/ | jq -r '.token')
+public_ecr_image_tags=$(curl -s -S  -k -H "Authorization: Bearer $public_ecr_image_tags_token"  'https://public.ecr.aws/v2/aws-observability/aws-for-fluent-bit/tags/list?page=1&page_size=250'  | jq -r '.tags[]' | sort -rV)
```

Comment:
Should make it `curl -s`? for silent

Reply:
How many tags do we have currently? Is hte page size a problem? I think i checked this before for docker hub and last winter we had something above 100...

Reply:
We checked and public ECR API call returns all tags (more than 250). Also, we only launched public ECR in Jan 2021, so there aren't all of our images in it. So at least for now, this works.